plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -665,6 +665,14 @@ private function get_paypal_order_shipping( $order ) {
 			return null;
 		}
 
+		// Make sure the country code is in the correct format.
+		if ( strlen( $country ) >= 3 ) {
```

Comment:
The if sentence should also use the `PAYPAL_COUNTRY_CODE_LENGTH` constant:
```suggestion
		if ( strlen( $country ) >= WC_Gateway_Paypal_Constants::PAYPAL_COUNTRY_CODE_LENGTH + 1 ) {
```

Reply:
Good idea. Applied it in https://github.com/woocommerce/woocommerce/pull/61678/commits/a22c14181d10a9550e3612faa74e74e1b77b901e